src/customEvent/rotate.js

```diff
@@ -17,13 +17,12 @@ eg.module("rotate", ["jQuery", eg, window, document], function($, ns, global, do
 	 *
 	 * @ko jQuery custom rotate 이벤트 지원
 	 * @name jQuery#rotate
-	 * @event
-	 * @param {Event} e event
-	 * @param {Boolean} e.isVertical vertical <ko>수직여부</ko>
+	 * @info
```

Comment:
@info dose not exist in jsdoc.

you should use jsdoc like this.
https://github.com/naver/egjs/blob/master/src/movableCoord.js#L281-L286